frappe/__init__.py

```diff
@@ -2022,8 +2022,10 @@ def logger(
 	)
 
 
-def log_error(message=None, title=_("Error")):
+def log_error(message=None, title=None):
 	"""Log error to Error Log"""
+	if title is None:
+		title = _("Error")
```

Comment:
```suggestion
def log_error(message=None, title="Error"):
	"""Log error to Error Log"""
```



I think we shouldn't translate while logging errors ever. Logging should be as simple as possible to avoid failures from dependency logging has.

This is why we removed translation wrappers in error logs in v14.